SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -1040,6 +1040,15 @@ SCDocHTMLRenderer {
 
 		this.renderHeader(stream, doc, body);
 		this.renderChildren(stream, body);
+
+		if(doc.isClassDoc) {
+			var moreExamples = "https://sccode.org/search?q=" ++ doc.klass;
+			stream << (
+				"</br>For community examples, please refer to &lt;<a href='"
+				+ moreExamples ++ "'>" ++ moreExamples ++ "</a>&gt;."
```

Comment:
what is the motivation to surround the link via `<` `>`?

Reply:
I thought this would make the web address stand out visually, like a small cosmetic change.
I'm not sure if it's really effective.

I can remove it if you don't like it.

Reply:
Done in https://github.com/supercollider/supercollider/pull/6985/commits/5be310e958764dfda92ec52c33b16e232a0d43ad!